pkg/controller/environment.go

```diff
@@ -72,8 +77,12 @@ func NewK8sEnvironment(config *ControllerConfig, log *logrus.Logger) (*K8sEnviro
 	if err != nil {
 		return nil, err
 	}
-
-	return &K8sEnvironment{kubeClient: kubeClient}, nil
+	log.Debug("Initializing snat client")
+	snatClient, err := snatclientset.NewForConfig(restconfig)
+	if err !=nil {
```

Comment:
The initialization is logged already. If there's an error with snatClient creation, it will show up in the logs through https://github.com/noironetworks/aci-containers/blob/403d6b9e0791de3c55718d24172758bb292af9c8/cmd/controller/main.go#L95